src/pages/organizers/[organizerId]/edit/index.page.tsx

```diff
@@ -18,6 +18,7 @@ export const getServerSideProps = getApplicationServerSideProps(
 
     return {
       props: {
+        organizer,
```

Comment:
Normally the line below should be enough, and then you use the same query hook to fetch the (cached already server fetched) organizer

Reply:
Sorry I see you're still in Draft. but normally you shouldn't have to pass the organizer prop separately 😉

Reply:
Sorry yes was leftover from local tests but the query is enough indeed